tests/test_profile.py

```diff
@@ -15,153 +15,101 @@
 
 class TestProfile:
 
+    @pytest.mark.xfail("'mozillians.allizom','mozillians.org' in config.getvalue('base_url')",
```

Comment:
This would return `('mozillians.allizom', False)` and so is not doing what you're expecting.